tests/testthat/test-lh-robust.R

```diff
@@ -0,0 +1,119 @@
+context("Estimator - lh_robust")
+set.seed(40)
+N <- 40
+dat <- data.frame(
+  Y = rnorm(N),
+  Y2 = rnorm(N),
+  Z = rbinom(N, 1, .5),
+  X = rnorm(N),
+  B = factor(rep(1:2, times = c(8, 12))),
+  cl = sample(1:4, size = N, replace = T),
+  w = runif(N)
+)
```

Comment:
You don't need to have perfect cluster or blocked variables (as is necessary in `difference_in_means`). I would just use `mtcars` for all of these tests.